File: Triangular Numbers/triangularnumbers.java
import java.util.Scanner;

public class triangularnumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        
        int number = 0;

        for (int i = 1; i <= n; i++) {
            number += i;
            System.out.println(number);
        }
    }
}

File: Triangular Numbers/triangularnumbers.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int number = 0;

    for (int i = 1; i <= n; i++) {
        number += i;
        printf("%d\n", number);
    }

    return 0;
}
